lib/audit/formIds.js

```diff
@@ -0,0 +1,39 @@
+const fs = require('fs');
+const glob = require('fast-glob');
+const yaml = require('js-yaml');
+
+const id = /(id: ?)(\w*)/;
+
+module.exports = {
+  audit: async () => {
+    let results = {};
+
+    // Collect ids.
+    const schemaFiles = await glob('{app,modules,custom_model_types}/**/model_schemas/*.yml');
```

Comment:
Hmmm, custom_model_types are inside app / modules, 

combinations should be more like

```
app
    model_schemas
    custom_model_types
modules
    my_module
        private
            model_schemas
        public
            custom_model_types
```

Same for form_configurations